reccmp/tools/aggregate.py

```diff
@@ -127,6 +129,15 @@ def main():
             "expected arguments for --output, --html, or --diff. (No output action specified)"
         )
 
+    if args.diff and len(args.diff) == 1 and not args.samples:
+        parser.error("expected two report files to diff")
```

Comment:
I'd make the error more actionable
```suggestion
        parser.error("--diff expects two report files to diff")
```